Scripts/script-DemistoLinkIncidents.yml

```diff
@@ -0,0 +1,44 @@
+commonfields:
+  id: DemistoLinkIncidents
+  version: -1
+name: DemistoLinkIncidents
+script: |-
+  var body = {
+      id: args.masterID,
+      otherIncidentsIDs: args.otherIDs.split(','),
+      removeLink: (args.unlink === 'yes')
+  };
+
+  var res = executeCommand('demisto-api-post', {url: 'incident/links', body: body});
```

Comment:
it is much more intuitive to use it like `/incident/links` (also RESTFul)
Please change in all places.